### DISCUSSION TITLE
[BUG] A video on home page it isn't visible from the mobile

### DISCUSSION THREAD

**Discussion by nicfab** [adityatelange/hugo-PaperMod#729] [Q&A]
I set a video (.mov or .mp4) in the home page.
I tried it in local, and it worked.
After the deployment, I could not see the video (it appears only on a coloured background).

The same is about the Twitter timeline that is an HTML code as follow (I changed the twitter username with user):

`<a class="twitter-timeline" data-width="600" data-height="800" data-dnt="true" href="https://twitter.com/user?ref_src=twsrc%5Etfw">Tweets by user</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
`
Are there issues with the mobile?

**Comment by nicfab** [comment]
Hi!
I had to change the video on the home page with an image.
However, you can test what I said, checking this webpage of my website here: https://www.nicfab.it/en/en/pages/t-t/
If you access that webpage from a mobile, you cannot see the Twitter timeline.
This is what happens to me.
I hope to receive a solution from you.

**Reply by adityatelange** [reply]
Hi Nic,

I can see that twitter timeline is visible properly.
![image](https://user-images.githubusercontent.com/21258296/149156475-da5c64d8-a987-4069-9040-87c086fe8725.jpg)

**Reply by nicfab** [reply]
> Hi Nic,
> 
> I can see that twitter timeline is visible properly. ![image](https://user-images.githubusercontent.com/21258296/149156475-da5c64d8-a987-4069-9040-87c086fe8725.jpg)

I don't know.
I am using an iPhone and here is the screenshot of what I see.
![PNG image](https://user-images.githubusercontent.com/24752671/149156894-284657a2-67bb-4990-ba7b-ad2208f64647.png)

**Comment by nicfab** [comment]
If I change the image on the home page with the video, can you check if you can see the video?
Probably it depends on Apple (possible?).

**Comment by nicfab** [comment]
I am so sorry!
I had set on my iPhone "Content Blocker" ON.
I deactivated the content blocker, and I see the Twitter Timeline correctly.

BtW, now on the home page, there is a video https://www.nicfab.it/en/
I can see the video from my Mac but not from my iPhone, and I asked someone to check also with Android.
**No one can see the video from mobile.**

**Reply by adityatelange** [reply]
Anyhow it is not theme related issue.

**Reply by nicfab** [reply]
> Anyhow it is not theme related issue.

What do you think is the issue?
